validation-test/Sema/type_checker_perf/fast/rdar18360240.swift.gyb

```diff
@@ -1,4 +1,4 @@
-// RUN: %scale-test --begin 2 --end 10 --step 2 --select incrementScopeCounter --exponential-threshold 1.5 %s
+// RUN: %scale-test --begin 2 --end 10 --step 2 --select incrementScopeCounter --polynomial-threshold 1.5 %s
```

Comment:
You need to remove the UNSUPPORTED below for this to actually run. (And also remove the FIXME note of course.)

Reply:
Thanks, @rudkx ! Sorry, I'm very clumsy today for some reason...